tools/src/org.graalvm.tools.lsp.test/src/org/graalvm/tools/lsp/instrument/TestInstrument.java

```diff
@@ -0,0 +1,40 @@
+package org.graalvm.tools.lsp.instrument;
```

Comment:
This should be in a `test` package, otherwise it's very confusing.
```suggestion
package org.graalvm.tools.lsp.test.instrument;
```